playkit/src/main/java/com/kaltura/playkit/player/vr/VRBarrelDistortionConfig.kt

```diff
@@ -0,0 +1,21 @@
+package com.kaltura.playkit.player.vr
+
+data class VRBarrelDistortionConfig(var paramA: Double = -0.068, // affects only the outermost pixels of the image
```

Comment:
lets give meaning full name for the outside wold maybe?

Reply:
Done.